tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -988,9 +987,19 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                               default=())
     # TODO: isn't it a weights precision type
     common_group.add_argument('--data_type',
-                              help=mo_convert_params['data_type'].description,
+                              help='[DEPRECATED] Data type for model weights and biases. '
+                                   'If original model is in FP32 and --data_type=FP16 is specified, '
```

Comment:
Please use the same formulation as in compress_to_fp32. There is no `model in FP32`, there are `FP32 weights` and other constants.

Reply:
Done.